build-release.sh

```diff
@@ -104,23 +105,40 @@ publish_nuget_packages() {
   done
 }
 
+can_publish_maven=0
+if [ ! -d "${basedir}/deps/keystore" ]; then
+  echo "Disabling Android library publishing as ${basedir}/deps/keystore doesn't exist."
+else
+  can_publish_maven=1
+fi
+
+publish_maven_library() {
+  if [ $can_publish_maven == 0 ]; then
+    return
+  fi
+  # FIXME: Might be worth reworking the script to make it all sudo-safe and use appropriate users throughout.
+  sudo sh build-android/upload-mavencentral.sh
+}
+
 godot_version=""
 templates_version=""
 do_cleanup=1
 make_tarball=1
 build_classical=1
 build_mono=1
 publish_nuget=0
+publish_maven=0
 
-while getopts "h?v:t:b:n-:" opt; do
+while getopts "h?v:t:b:p:n-:" opt; do
   case "$opt" in
   h|\?)
     echo "Usage: $0 [OPTIONS...]"
     echo
     echo "  -v godot version (e.g: 3.2-stable) [mandatory]"
     echo "  -t templates version (e.g. 3.2.stable) [mandatory]"
     echo "  -b build target: all|classical|mono|none (default: all)"
-    echo "  -n publish nuget packages (default: false)"
+    echo "  -p publish target: all|nuget|maven|none (default: none)"
+    echo "  -n publish nuget packages (default: false; legacy option, use -p instead)"
```

Comment:
I think we can drop `-n`, this is only used by us, we can adapt to the new syntax (as long as using `-n` properly errors out, so we don't do mistakes when changing branches, etc.).

Reply:
Looks like it should error out, so removed!